config/config.yml.sample

```diff
@@ -1,5 +1,7 @@
 defaults: &defaults
 
+  edit_password_url: url
```

Comment:
@yunixon looks like there should be some host with url. It's not obvious from you example. Please make better example.